tutor/src/components/annotations/sections-filter.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import Multiselect from '../multi-select';
```

Comment:
I'm glad there was an existing widget for this.

Reply:
Yeah, sorry for not mentioning it earlier, I'd kind of forgotten about it as well